core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -951,6 +1083,258 @@ export class LoggedOutUser extends BaseUser {
+      'Android'
+    );
+  }
+
+  /**
+   * Function to click the Visit Classroom button in the Teach page
+   * and check if it opens the Math Classroom page.
+   */
+  async clickVisitClassroomButtonInTeachPage(): Promise<void> {
+    const visitClassroomButtonInTeachPage = this.isViewportAtMobileWidth()
+      ? visitClassroomMobileButtonInTeachPage
+      : visitClassroomDesktopButtonInTeachPage;
+    await this.clickButtonToNavigateToNewPage(
+      visitClassroomButtonInTeachPage,
+      'Visit Classroom button',
+      mathClassroomUrl,
+      'Math Classroom'
+    );
+  }
+
+  /**
+   * Function to click the Browse Library button in the Teach page
+   * and check if it opens the Community Library page.
+   */
+  async clickBrowseLibraryButtonInTeachPage(): Promise<void> {
+    const browseLibraryButtonInTeachPage = this.isViewportAtMobileWidth()
+      ? browseLibraryMobileButtonInTeachPage
+      : browseLibraryDesktopButtonInTeachPage;
+    await this.clickButtonToNavigateToNewPage(
+      browseLibraryButtonInTeachPage,
+      'Browse Library button',
+      communityLibraryUrl,
+      'Community Library'
+    );
+  }
+
+  /**
+   * Function to click the Browse Our Lessons button in the Teach page
+   * and check if it opens the Math Classroom page.
+   */
+  async clickExploreLessonsButtonInTeachPage(): Promise<void> {
+    const exploreLessonsButtonInTeachPage = this.isViewportAtMobileWidth()
+      ? exploreLessonsMobileButtonInTeachPage
+      : exploreLessonsDesktopButtonInTeachPage;
+    await this.clickButtonToNavigateToNewPage(
+      exploreLessonsButtonInTeachPage,
+      'Explore Lessons button',
+      mathClassroomUrl,
+      'Math Classroom'
+    );
+  }
+
+  /**
+   * Function to click the Partner With Us button in the Partnerships page
+   * and check if it opens the Partnerships Google form.
+   * The button is in the first section of the page.
+   */
+  async clickPartnerWithUsButtonInPartnershipsPage(): Promise<void> {
+    await Promise.all([
+      this.page.waitForNavigation(),
+      this.clickOn(partnerWithUsButtonAtTheTopOfPartnershipsPage),
+    ]);
+
+    const allowedUrls = [
+      partnershipsFormShortUrl,
+      partnershipsFormUrl,
+      `${partnershipsFormUrl}?usp=send_form`,
+    ];
+
+    // The Google Form URL changes from the 1st to the 2nd and from 2nd to the
+    // 3rd in a short span of 500-1000 ms for it's own reasons which we can't
+    // control.So we need to check for all the 3 URLs as all of them are valid.
+    expect(allowedUrls)
+      .withContext(
+        'Partner With Us button at the top of the Partnerships page should open' +
+          'the Partnerships Google Form page'
+      )
+      .toContain(this.page.url());
+  }
+
+  /**
+   * Function to change the site language to the given language code.
+   * @param langCode - The language code to change the site language to. Example: 'pt-br', 'en'
+   */
+  async _changeSiteLanguage(langCode: string): Promise<void> {
```

Comment:
Why is there an underscore at the start of the function name? If you want to make this a private function, I believe you should add the keyword `private` before `async`.